controllers/control_plane_init_locker_test.go

```diff
@@ -32,6 +32,42 @@ import (
 )
 
 func TestControlPlaneInitLockerAcquire(t *testing.T) {
+	tests := []struct {
+		name     string
+		getError error
+	}{
+		{
+			name:     "create succeeds",
+			getError: apierrors.NewNotFound(schema.GroupResource{Group: "", Resource: "configmaps"}, "uid1-configmap"),
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			l := &controlPlaneInitLocker{
+				log: log.ZapLogger(true),
+				configMapClient: &configMapsGetter{
+					getError: tc.getError,
+				},
+			}
+
+			cluster := &clusterv2.Cluster{
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: "ns1",
+					Name:      "name1",
+					UID:       types.UID("uid1"),
+				},
+			}
+
+			acquired := l.Acquire(cluster)
+			if !acquired {
+				t.Fatal("acquired was false but it should have been true")
+			}
+		})
+	}
+}
+
+func TestControlPlaneInitLockerAcquireErrors(t *testing.T) {
```

Comment:
[1]

Reply:
This one has 3 table tests, I don't think we should get rid of subtests.

Reply:
time to take a break...